projects/angular/src/popover/dropdown/STYLES.md

```diff
@@ -0,0 +1,42 @@
+# Dropdown
+
+## Variables
+
+| Variable                           | Description                                  |
+| ---------------------------------- | -------------------------------------------- |
+| `$dropdown-menu-background-color`  | Background color of the dropdown menu.       |
+| `$dropdown-item-padding`           | Padding applied to the dropdown items.       |
+| `$dropdown-item-hover-background`  | Background color when hovering over an item. |
+| `$dropdown-item-active-background` | Background color of the active item.         |
+| `$dropdown-item-active-color`      | Text color of the active item.               |
+
+## Classes
+
+| Class                   | Description                                               |
+| ----------------------- | --------------------------------------------------------- |
+| `.dropdown`             | Styles the overall dropdown component.                    |
+| `.dropdown-toggle`      | Styles the toggle element inside the dropdown component.  |
+| `.btn`                  | Styles the button elements inside the dropdown component. |
```

Comment:
This is coming from the trigger button, it's not specific to the dropdown. Even if some changes are applied to this in the dropdown CSS, the users should identify it and change it as part of the button element. Please remove.